docs/ops.md

```diff
@@ -69,6 +69,10 @@ This is not what we want.
 As ops, we want to be able to upgrade containerd and allow existing containers to keep running without interruption.
 Setting `KillMode` to `process` ensures that systemd only kills the containerd daemon and not any child processes such as the shims and containers.
 
+The following `systemd-run` command is equivalent to the changes in the service file:
```

Comment:
Maybe say similar instead of equivalent because we are missing the limits on fds and such right?

Reply:
Right. Updating.